lizmap/test/data/print.qgs.cfg

```diff
@@ -0,0 +1,178 @@
+{
+    "metadata": {
+        "qgis_desktop_version": 32214,
+        "lizmap_plugin_version_str": "3.9.11-alpha",
+        "lizmap_plugin_version": 30911,
+        "lizmap_web_client_target_version": 30700,
+        "lizmap_web_client_target_status": "Dev",
+        "instance_target_repository": "",
+        "instance_target_url": "http://lizmap.local:8130/",
+        "project_valid": true
+    },
+    "warnings": [],
+    "options": {
+        "projection": {
+            "proj4": "+proj=longlat +datum=WGS84 +no_defs",
+            "ref": "EPSG:4326"
+        },
+        "bbox": [
+            "3.8331346980485472",
+            "43.5561629722011",
+            "3.920928392916381",
+            "43.645371064914904"
+        ],
+        "mapScales": [
+            10000,
+            25000,
+            50000,
+            100000,
+            250000,
+            500000
+        ],
+        "minScale": 10000,
+        "maxScale": 500000,
+        "initialExtent": [
+            3.8331346980485472,
+            43.5561629722011,
+            3.920928392916381,
+            43.645371064914904
+        ],
+        "popupLocation": "dock",
+        "pointTolerance": 25,
+        "lineTolerance": 10,
+        "polygonTolerance": 5,
+        "tmTimeFrameSize": 10,
+        "tmTimeFrameType": "seconds",
+        "tmAnimationFrameLength": 1000,
+        "datavizLocation": "right-dock",
+        "theme": "dark",
+        "fixed_scale_overview_map": true
+    },
+    "layers": {
+        "legend-expanded": {
+            "id": "points_2adbbf7e_53dd_49bd_a1dc_2dbfe172b757",
+            "name": "legend-expanded",
+            "type": "layer",
+            "geometryType": "point",
+            "extent": [
+                1.7976931348623157e+308,
+                1.7976931348623157e+308,
+                -1.7976931348623157e+308,
+                -1.7976931348623157e+308
+            ],
+            "crs": "EPSG:4326",
+            "title": "legend-expanded",
+            "abstract": "",
+            "link": "",
+            "minScale": 1,
+            "maxScale": 1000000000000,
+            "toggled": "False",
+            "popup": "False",
+            "popupFrame": null,
+            "popupSource": "auto",
+            "popupTemplate": "",
+            "popupMaxFeatures": 10,
+            "popupDisplayChildren": "False",
+            "popup_allow_download": true,
+            "legend_image_option": "expand_at_startup",
+            "groupAsLayer": "False",
+            "baseLayer": "False",
+            "displayInLegend": "True",
+            "group_visibility": [],
+            "singleTile": "True",
+            "imageFormat": "image/png",
+            "cached": "False",
+            "serverFrame": null,
+            "clientCacheExpiration": 300
+        }
+    },
+    "atlas": {
+        "layers": []
+    },
+    "locateByLayer": {},
+    "attributeLayers": {},
+    "tooltipLayers": {},
+    "editionLayers": {},
+    "layouts": {
+        "list": [
```

Comment:
Yes, we must have something.
The JSON is auto-generated following the same template.
* `layout` is the name of the panel.
* `list` in this case means the main table, with edit, move up, move down, (create/remove for other tables as well)
* `config` is some settings in the panel, not in the main table above.

Similar to the `filter by polygon` : 

```json
    "filter_by_polygon": {
        "config": {
            "filter_by_user": false
        },
        "layers": []
    },
```

If one day we add a new **global** setting in the **layout** panel, it will land 

```json
    "layouts": {
        "config": [...
```